apps/frontend/src/app/5_pages/MarketMakingPage/components/PoolsTable/components/PoolsTableAction/PoolsTableAction.tsx

```diff
@@ -76,7 +88,7 @@ export const PoolsTableAction: FC<PoolsTableActionProps> = ({ pool }) => {
                 dataAttribute="pools-table-deposit-button"
                 className="w-full lg:w-auto prevent-row-click"
                 disabledStyle={actionLocked}
-                disabled={!account}
+                disabled={!account || isMynt}
```

Comment:
Shouldn't the `MYNT` pool be visible only if I have some balance in it?

<img width="1194" alt="image" src="https://github.com/DistributedCollective/sovryn-dapp/assets/14878713/20816c17-9ac4-44f9-9bec-920e6ee044dd">